src/Plugins/ImageProcessor.Web/ImageProcessor.Web.Plugins.AzureBlobCache/AzureBlobCache.cs

```diff
@@ -287,7 +302,9 @@ public AzureBlobCache(string requestPath, string fullPath, string querystring)
                 {
                     string container = RemoteRegex.Replace(this.cloudSourceBlobContainer.Uri.ToString(), string.Empty);
                     string blobPath = RemoteRegex.Replace(this.RequestPath, string.Empty);
-                    blobPath = blobPath.Replace(container, string.Empty).TrimStart('/');
+                    blobPath = blobPath.Replace(container, string.Empty).TrimStart('/')
+                        //.Replace("127.0.0.1_10000", "127.0.0.1:10000")
+                        .FixDevelopmentStorageEmulatorUrl();
```

Comment:
I definatly have to read more about this async stuff. Currently I am not very familiar with it. But your project is a good starting point and inspiration! Thank you very much! :+1: